documentation/how-to/map-interaction.en.md

```diff
@@ -24,6 +24,18 @@ On double-tap or long-press on a layer, you get offered the following possibilit
 - *Reload data* to get the current data of a layer with remote sources.
 - *Show feature list* to show all the layer's features in the identification list.
 
+## Sort layer features
+
+If you'd like to configure the order of features in 'Show feature list' in QField, you have the following options to preconfigure this on QGIS:
```

Comment:
```suggestion
## Sort layer features
:material-monitor: Desktop preparation

If you'd like to configure the order of features in 'Show feature list' in QField, you have the following options to preconfigure this on QGIS:
```

We can add something like this, I think it is nice hint that you cannot change it on QField.

https://github.com/opengisch/QField-docs/blob/master/documentation/how-to/pictures.en.md?plain=1#L20